ui/pages/swaps/swaps.util.js

```diff
@@ -418,21 +416,27 @@ export async function fetchSwapsFeatureFlags() {
   return response;
 }
 
-export async function fetchSwapsQuoteRefreshTime(chainId, useNewSwapsApi) {
+export async function fetchSwapsRefreshRates(chainId, useNewSwapsApi) {
```

Comment:
My main concern here is that this method is declared here but only ever consumed in the controller. Could we lift this into the controller itself and import the pieces it requires from this file? It isn't a complete fix but it's better than so vastly separating implementation with the intended call site.

Reply:
That's a great point! Basically, we don't need it in utils if it's only used once and that code can be in the Swaps controller instead. I don't see the `getBaseApi` nor `fetchWithCache` function calls in the Swaps controller, but we can start doing it now for this one.